test/InquirerUnitTest/Validators/CreditCardNumberValidatorUnitTest.cs

```diff
@@ -0,0 +1,62 @@
+﻿using FluentAssertions;
+using InquirerCore.Validators;
+using Xunit;
+
+namespace InquirerUnitTest.Validators
+{
+    public class CreditCardNumberValidatorUnitTest
+    {
+        [Theory]
+        [InlineData("4556737586899855")]
+        [InlineData("4929 9222 5630 0976")]
+        [InlineData("4929-9222-5630-0976")]
+        [InlineData("4929922256300976")]
+        [InlineData("5522277758452967")]
+        [InlineData("4614173898370866")]
+        [InlineData("5345422286958722")]
+        [InlineData("343850055763282")]
+        [InlineData("345964336736377")]
+        [InlineData("343540876891665")]
+        [InlineData("343850055763282")]
+        [InlineData("6304364589469402")]
+        [InlineData("36452061374162")]
+
+        public void ShouldAcceptValidCreditCard(string validCreditCard)
+        {
+            var validator = new CreditCardNumberValidator();
+
+            var isValid = validator.Validate(validCreditCard);
+
+            isValid.Should().BeTrue();
+        }
+
+        [Theory]
+        [InlineData("111a-2222-3333-4444")]
+        [InlineData("1111/2222 3333 4444")]
+        [InlineData("1111222233334443")]
+        [InlineData("1111 2222 3333 4443")]
+        [InlineData("1111-2222-3333-4443")]
+        [InlineData("11112123123123123123123122233334444")]
+        [InlineData("12345678901234567890")]
+        [InlineData("213@1")]
+        [InlineData("2131")]
+        [InlineData("123456789012345'")]
+        [InlineData("asdfghjkl;qwerty")]
+        public void ShouldNotAcceptInvalidCreditCard(string validCreditCard)
```

Comment:
As this validator has some complex rules, I think that break the test for invalid specific cases would be better. Like:
 - One for invalid size
 - One for invalid chars
 - One for invalid Luhn check

this would make the test more readable and easier to understand all the invalid cases

Reply:
Good point, thank you for you feedback.
Working on it.